config/overlays/openshift/manager-patch.yaml

```diff
@@ -0,0 +1,11 @@
+- op: add
+  path: "/spec/template/spec/containers/0/args"
+  value:
+    - --metrics-addr=127.0.0.1:8080
```

Comment:
Do we want to enable the RBAC creation flag as well?

Reply:
That flag does nothing now.